final_project/snapvisite/templatetags/custom_tags.py

```diff
@@ -9,3 +9,16 @@ def all_categories():
     return Category.objects.all()
 
 
+@register.filter('duration_format')
+def duration_format(value):
+    h = 'h'
+    m = 'min'
```

Comment:
jaki sens to do zmiennej przypisywać jeśli to i tak potem leci do stringa? Tymbardziej, że nazwy nie są bardziej opisowe niż wartość